composer.json

```diff
@@ -10,9 +10,10 @@
     },
     "require": {
         "php": "^7.4.7",
+        "composer/package-versions-deprecated": "^1.10.99",
```

Comment:
Note: we're already on `^7.4.7`, so a dependency change shouldn't be needed here

Reply:
It's needed because phly/keep-a-changelog uses it; otherwise, there are conflicts.

Reply:
Yeah, something messed up with that replacement. Had some weird problems when using laminas-cli aswell...
Not sure what causes those conflicts...

Reply:
Could be any of the CLI tools - not a big deal though, as we're free to use `composer/composer:^2` here (it's an application, not a library)